course-details.md

```diff
@@ -1,9 +1,43 @@
-This course is about continuous delivery, or CD, with GitHub Actions. We create two workflows to deploy our app to an AWS S3 bucket automatically. CD is an important part of modern software development, and it has a big meaning. CD is the practice of delivering software faster and with higher quality. To do this, it takes many different practices, behaviors, and technologies. 
+This course is about continuous delivery, or CD, with GitHub Actions. CD is a crucial part of modern software development, and it can have a monumental impact on development projects. 
```

Comment:
@hectorsector @lmkeston Let's keep in mind that this course is going to be a part of that learning path which is a crucial part of the Migrating to Actions offering.

The course isn't necessarily about CD, it's more about using GitHub Actions for CD.  I think it's important to keep in mind that it's a course about GitHub Actions that covers CD vs a course on CD that happens to use GitHub Actions.

Reply:
> The course isn't necessarily about CD, it's more about using GitHub Actions for CD. I think it's important to keep in mind that it's a course about GitHub Actions that covers CD vs a course on CD that happens to use GitHub Actions.
> 
> This thought also applies to further pieces of the description.

Great point @mattdavis0351, I'll work on tightening this up tomorrow